src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/LLVMObjectWriter.cs

```diff
@@ -814,10 +814,17 @@ private void GetCodeForExternMethodAccessor(ExternMethodAccessorNode node)
             LLVMValueRef externFunc = externFuncModule.AddFunction(externFuncName, externFuncType);
 
             // Add import attributes if specified.
-            if (_compilation.ConfigurableWasmImportPolicy.TryGetWasmModule(externFuncName, out string wasmModuleName))
+            // Copy the prefix from WasmConfigurablePInvokePolicy here to avoid making the policy classes public.
+            if (node.Signature != null && externFuncName.StartsWith("__wasm_import_"))
             {
-                externFunc.AddFunctionAttribute("wasm-import-name", externFuncName);
-                externFunc.AddFunctionAttribute("wasm-import-module", wasmModuleName);
+                foreach (MethodDesc method in node.EnumerateMethods())
```

Comment:
Perf nit: adding a `GetSingleMethod()` to the accessor node, like:
```
_methods switch { MethodDesc methods => method, _ => /* foreach loop over HashSet<MethodDesc> */ }
```
will avoid allocating the enumerator.

Reply:
Thanks, I did the `switch` more like the one earlier in the file as I couldn't see how to do the foreach with the other style as I think it needs an expression.
```cs
switch (_methods)
{
    case MethodDesc oneMethod:
        return oneMethod;

    default:
        foreach (MethodDesc method in (HashSet<MethodDesc>)_methods)
        {
            return method;
        }

        return null;
}
```